### DISCUSSION TITLE
Icon Request: fa-troll

### DISCUSSION THREAD

**Discussion by ghost** [FortAwesome/Font-Awesome#21165] [Icon Requests]
Request for fa-troll (internet troll) icon.

![fa-troll](http://i.imgur.com/KFIeIVq.png)

Wikipedia: https://en.wikipedia.org/wiki/Internet_troll

**Comment by tblank555** [comment]
I second this request!

**Comment by legendarydrew** [comment]
Third-did!

**Comment by ghost** [comment]
Could be useful in places like comments, forum posts.

**Comment by dumptyd** [comment]
I fourth this.

**Comment by pierre-tranchard** [comment]
👍  i stand for this too
